provisions/roles/openshift/defaults/main.yml

```diff
@@ -17,7 +17,7 @@ openshift_shared_dirs:
 openshift_volumes:
   - /:/rootfs:ro
   - /var/run:/var/run:rw
-  - /sys:/sys
+  - /sys:/sys:rw
```

Comment:
should this change be applicable only when `deployment==docker`?

Reply:
this actually doesn't matter as by default it's read/write for volumes. the only reason why I put `:rw` is to make it consistent with the rest of the volumes.